porting/halium_7-1/Configuring.rst

```diff
@@ -118,6 +118,218 @@ At the end of the file, append this::
+
+Although many of these settings will be collected in one place in your defconfig file, some will likely be in other places as well, making it difficult to ascertain all necessary settings. The directions below may help.
+
+After completing a build of halium-boot.img, go to your ``out/target/product/[device]/obj/KERN_OBJ`` directory. There, run the following command::
+
+    ARCH=arm64 make menuconfig
+
+If your device is armhf, use ``ARCH=arm`` instead.
+
+This will bring up menuconfig complete with the defconfig settings from your build. You then navigate to the bluetooth drivers submenu and browse through all activated settings, recording which ``CONFIG_xxxxx`` settings apply to those that are activated for your device, as well as the information about what this setting does and which other settings it depends upon (found under Help). Save this information for later reference.
+
+Download the backport scripts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Clone the backport scripts into a directory outside your halium source tree by issuing this command from your home (~) directory::
+
+    git clone https://github.com/ubuntu-phonedations/backports.git -b for-ubuntu backport-scripts
+
+This downloads the backport scripts prepared by Canonical based on the :ref:`original Backports Project <Backports>` mentioned above, and places them in the directory ``~/backport-scripts``. The scripts are specifically written to backport from kernel version 4.2.
+
+Download kernel source to backport from
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a directory (outside your halium source tree) for the kernel source from which you will pull the newer drivers::
+
+    mkdir ~/kernel-backports
+
+Now clone the kernel source for the version and branch you need (v4.2 in the example below) into this directory::
+
+    cd ~/kernel-backports
+    git clone https://kernel.googlesource.com/pub/scm/linux/kernel/git/next/linux-next -b v4.2
+
+.. Note::
+
+    Although there are other kernel versions besides v4.2 available (as witnessed by available version tags on `the webpage <https://kernel.googlesource.com/pub/scm/linux/kernel/git/next/linux-next>`_), the backport script is specifically tailored to backporting from version 4.2 and thus effectively limits you to this option.
+
+Run script and fix errors
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Navigate to your backport scripts directory and issue the command below (using Python2 as shown)::
+
+    python2 ./gentree.py --copy-list ./copy-list --integrate --clean --git-revision v4.2 ~/kernel-backports/linux-next ~/halium/kernel/[VENDOR]/[MODEL_NAME]
+
+You will get error messages at the end and if they do not mention the backport Makefile and/or Kconfig, this means that the script has aborted before completion. You will then need to determine the cause and retry. 
```

Comment:
This is a very roundabout way of saying it. How about:

It is to be expected that there are errors during this step. You will then have to determine the cause, fix it and retry. The last error message concerns the Makefile ....